.github/workflows/pr.yml

```diff
@@ -168,6 +168,20 @@ jobs:
       contents: read
     uses: ./.github/workflows/verify-devcontainers.yml
 
+  verify-codegen:
+    name: Verify Codegen in libcudacxx
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Run libcudacxx codegen verification
+        id: verify-codegen
+        run: |
+          sudo apt-get update
+          sudo apt-get install ninja-build
+          export CXX="g++"
+          ./ci/verify_codegen.sh
```

Comment:
No need to change, but could we just run this in one of the devcontainers?

Reply:
It'd need to download the devcontainer, and we don't need the CTK. Running the verification steps locally are basically instant because everything is turned off.